py3status/modules/static_string.py

```diff
@@ -3,35 +3,31 @@
 Display static text.
 
 Configuration parameters:
-    format: text that should be printed (default '')
+    format: default format for this module (default 'hello, world!')
```

Comment:
Oh. I don't even notice the difference between... `display format for this module` and... `default format for this module`. I'll change that now. Cheers. 👍